### ISSUE TITLE
Discussion on “Command Center” naming

### ISSUE THREAD

**Issue by rmartinezduque** [WordPress/gutenberg#50925]
## Context
Gutenberg 15.6 introduced a new feature called "[Command Center](https://href.li/?https://github.com/WordPress/gutenberg/issues/48457)" under the experimental flag that aims to serve different purposes:

- Quickly search, navigate, and switch between different types of content
- Run commands to perform tasks or actions
- Extend and customize the tool, also with AI, via third parties (plugins)

For now, jumping into different site content and sections seems like the primary use case for users, while executing actions will be a more advanced function for them.

The concept of a command center can convey the idea of a centralized location to execute commands and manage tasks, but it seems a bit technical and carries some militant connotations. Additionally, after reading some [feedback left in Riad’s call for feedback](https://make.wordpress.org/core/2023/05/17/command-center-request-for-feedback/), my impression (from a marketing perspective) is that this name may not fully convey its potential and different use cases.

## Proposed Solution/Name
Consider giving this feature a name that better captures its different use cases, helps work on messaging that reflects benefits, and appeals to a less technical audience.

Taking into account factors such as functionality, resonance (target audiences), and distinctiveness, I’d like to suggest “Wayfinder” as a potential new name.

"[Wayfinder](https://en.wiktionary.org/wiki/wayfinder)" refers to a person, a route, or a means to achieve something. It’s flexible and can better capture this feature's different possibilities, unlike other names that may be more descriptive of a single aspect.

## Global Considerations
As a non-native English speaker and considering the project's global nature, I’m aware that it may be challenging to translate this word into other languages since it may not have a direct literal translation. That is, it may be translated into two or more words, but in a way, it can lose its native meaning.

Somehow, I think this is a unique (enough) name similar to “plugin” or “Gutenberg” in the sense that it may not require localized names, but all and any diverse input/considerations will be welcome.

**Comment by annezazu** [comment]
I dig Wayfinder, especially thinking about the longer term aims, and definitely want to be mindful of global considerations. Thanks for kicking off this discussion so thoughtfully.

**Comment by eidolonnight** [comment]
I also support "Wayfinder" as a good and memorable name. Considering the ambitions for this, it's future-proof enough to encompass all the planned functionality.

**Comment by courtneyr-dev** [comment]
I'm thinking on what a new user would search for that should result in materials about this feature.  First Result for "WordPress Wayfinder" is this plugin that I use and enjoy: [https://wordpress.org/plugins/wayfinder)](https://wordpress.org/plugins/wayfinder)
Wayfinder as a search term results in lots of different results, from videogames to homegoods.

**Comment by rmartinezduque** [comment]
Yes, I imagine finding a word that isn't used in other contexts is difficult, and I wonder if that would also mean ranking better in search results for any related material, documentation, etc. For example, if I do a search for "WordPress dashboard," I see a WordPress.com article and [this plugin](https://wordpress.org/plugins/ultimate-dashboard/) in the first results. In any case, I agree that it's definitely good to keep this in mind. Thanks, Courtney!

**Comment by jomarieminney** [comment]
While I quite like Wayfinder I agree it would be confusing given that there is already a plugin with that name. I do agree with the comments on Command Center being very military, but I think it would be something that would be helpful to get feedback on from Polyglots about whether that is definitely an issue. If the primary purpose of the feature is to run commands it does make sense to call it that, but if that is only part of what it does (and say, navigation, is equally important) here are some alternatives that might be worth considering:

- Control Centre (or Center, depending on localisation!)
- Control Hub (Similar to the first one but without the center/centre issue that comes with localisation)
- WordPress Explorer (similar meaning to Wayfinder? There is a WP Explorer blog, but no plugins of this name, although some do contain the word 'explorer'.)
- WordPress Spotlight (definition: the state of being prominent: widely known or eminent. verb. illuminate)

I would point out that while the navigation aspect that was mentioned in the comments is cool and potentially useful, it sounds like the primary purpose of the feature is still to execute commands. My vote would probably be for 'Command Hub' or 'Control Hub' mainly because as an Australian person the word 'Center' makes me cringe whenever I see it written 🙃

**Comment by kaitohm** [comment]
I only speak English and Japanese, but here is my input.

While Wayfinder is a cool term, it doesn't exist in Japanese. Google results for the term in Japanese return a Star Wars reference as the first result. If we adopt this, we'd probably use a phonetical translation of the word (like we do for plugin), and have to teach the meaning of it to folks.

I first liked "Control Center" or "Control Hub". But a potential Japanese translation of the word "control" could be "管理", which is the exact word we're already using for wp-admin: `管理画面`

So of all the suggestions here, the original "Command Center" actually feels the best...? Japanese are also familiar with "Explorer" from Internet Explorer days, and "spotlight" is a recognizable term, too.

**Comment by lidialab** [comment]
From my understanding the name of this function can be similar to something like the "Commands Palette" in VS Code or Sublime Text editors (and in that case "Palette" for us it will probably be translated in the same way as we would translate "command centre").

"Wayfinder" is distinctive, but a little tricky to translate. As an Italian speaker, I think that the Italian Polyglots would consider to let it the same as in the original, or to find a very alternative translation. For example when considering "Finder" as a "seeker", we often hesitate to attribute human characteristics to machines.

BUT it's a very complicate path to find the right way to name something new :) , so if this is the way, we will find our own to translate it ;)

I really appreciate that it was thought to involve Polyglots, we will have the task of translating, but I also think the main concern is to avoid "red flags", like the military connotations. What Polyglots really need to know are always the context and the possible evolutions.

I'm not sure if some of these could fit, but it's just to try to add some ideas:
Actions hub
Quick commands
Quick actions

**Comment by rmartinezduque** [comment]
Thank you, @jomarieminney, @bsanevans, and @lidialab for your feedback!

> BUT it's a very complicate path to find the right way to name something new :) , so if this is the way, we will find our own to translate it ;)

It is, indeed! Especially finding a name that works well on a marketing level and also translates well in all WordPress languages. But I appreciate the input and willingness to collaborate :)

From what I'm hearing, there does not seem to be any red flags with "Wayfinder," other than it's tricky to translate as it might not exist in some languages (I wonder if that opens the doors to treat it similarly to the word "plugin"). I see other suggested names may also have similar challenges. E.g., the word "Control" may have different meanings and also not translate well into other languages, and "Explorer", "Spotlight", and "Command Palette" have other concepts or product/plugin names associated with it.

In my opinion, it might be worth trying this name (Wayfinder) and keep learning what other factors might be good to consider when naming some features. If something comes up, I guess we can always discuss it further and look for alternatives.

**Comment by megane9988** [comment]
I don't want to confuse the discussion here. However, as @bsanevans  wrote, as a Japanese person, I honestly heard the word `Wayfinder` for the first time.
And when I searched, the title of the game was hit. As a result, I wondered if Wayfinder had such a heroic image.

How about a `quick finder`?

**Comment by jomarieminney** [comment]
I think the comment from Courtney about the existing plugin of the same name is valid. It would be a bit of a kick in the face to the plugin developers... and potentially confusing for users searching the repository?

**Comment by rmartinezduque** [comment]
Thanks for your feedback, @megane9988. Personally, I also like Quick Finder, Pathfinder, or Shortcut Finder as alternatives that can communicate a similar meaning or concept if we don't want to consider Wayfinder finally.

**Comment by rmartinezduque** [comment]
@jomarieminney, I think all feedback is valid. And Courtney brought up a good point, but in my opinion, it doesn't seem like a blocker or a red flag in this case.

> and potentially confusing for users searching the repository?

I'm not sure I'm following you or understanding what you mean. This feature can be accessed by typing a keyboard shortcut in the Site Editor interface. I don't see how it can have an impact on the repository search.

**Comment by annezazu** [comment]
For extra context around how this feature is already evolving, it's been included in the Title Bar of the Site Editor and is invoked when selected: https://github.com/WordPress/gutenberg/pull/50369 I imagine it will be surfaced in other UIs for improved discoverability.

**Comment by JosVelasco** [comment]
My vote is for Finder. Or Site Finder. Boring but easy to... find.

**Comment by aurooba** [comment]
Navigation is an action. And we know there will likely be other actions you can take from this feature as well, some may navigate you to a different page and some may affect things on the current page. 

It's also a feature that let's you mix and match things, modify options, and see what's possible.
 
I think Wayfinder sounds cool, but puts more emphasis on navigation than anything else.

I also think we should avoid other languages having to invent a word or find convuluted alternatives, as much as possible.

In Urdu and Hindi for example, it translates to a 4 word phrase, although in both those languages it's likely that it would just stay the actual word.

I would prefer to see a name like Actions Palette (for example in Urdu and Hindi, the word palette already exists and its association with choosing colours - both set and custom, lends itself well to the idea of taking action).

**Comment by eidolonnight** [comment]
Another consideration with naming, especially in this case, is _where_ the name will be used. In the case of this feature, at least according to [the mockups I've seen so far](https://github.com/WordPress/gutenberg/issues/48457#issuecomment-1487439383), the name does not appear within the interface. The name, like Gutenberg, will be used primarily for marketing and documentation. That influences naming and translation concerns.

Even differences in US vs UK English make it extremely difficult to find a catchy (Ie. marketable)and universally understood (Ie. easily documented) name. That's why I agree with the aforementioned recommendation that we treat this a bit like "Gutenberg" or "plugin". If this were appearing frequently within the interface then I would perhaps feel different.

**Comment by JustinSainton** [comment]
For reasons unknown even to myself, anytime I've used or built these, I've always called them Command Palettes.
 🎨

**Comment by lidialab** [comment]
To satisfy our curiosity...
Also the Windows Terminal has its "Command Palette".
Microsoft has translated it in Italian "Riquadro comandi", something like "Commands box"/"Commands frame".

And there is a site: https://www.commandpalette.org/
I choose to read this https://www.commandbar.com/blog/command-palette-past-present-and-future, but in that site there are many articles.

They can be inspirational and there is also a useful list of shortcuts used by those programs. (Combo with K or P are the preferred ones.)

**Comment by rosswintle** [comment]
There's two things here. One is "What is it?" and the other is "What is it called?"

I would rather it was just called a command palette on both counts. This is by far the most common term in use to describe this kind of thing. I see no need to stray from popular convention. Anything else is either confusing or marketing and I don't like either.

It's a command palette. Can we call it a command palette please?

Some research - 50% are "Command Palette". The others are unique. "Launcher" occurs twice but I don't think that applies here.

Sublime Text: "Command Palette" (https://docs.sublimetext.io/reference/command_palette.html)

VS Code: "Command Palette" (https://code.visualstudio.com/api/ux-guidelines/command-palette)

GitHub: "Command Palette" (https://docs.github.com/en/get-started/using-github/github-command-palette)

Figma: "Quick Actions Bar" (https://help.figma.com/hc/en-us/articles/360040328653-Use-shortcuts-and-quick-actions)

Alfred doesn't say what sort of "thing" it is.

Spotlight is "Spotlight" (probably (TM))

Raycast calls itself a "Launcher". It has a "Search bar" and and "Action Panel"

Wox - which comes up in Alfred for Windows searches - also calls itself a "Launcher"

PHPStorm calls it a "Search Window"

MS Edge has a "Command Palette"

Jira has a "Command Palette"

Descript has "Conductor" but it's described as a "Command Palette" (https://www.descript.com/blog/article/overdub-stock-voices)

Superhuman has "Superhuman command" which is also described as a "Command Palette" (https://blog.superhuman.com/how-to-build-a-remarkable-command-palette/)

Tower has "Quick open" (https://www.git-tower.com/blog/mastering-tower)

**Comment by rmartinezduque** [comment]
To help wrap up the discussion and based on the above comments, I note some preference for using the term "Command Palette" due to its common use in the software industry. More alternative names are also mentioned. I see this point as completely valid and understand the reasons, although I think it doesn't help address one of the comments I made when opening the issue about appealing to a less technical audience.

The term "Command Palette" focuses more on the technical aspect, emphasizing its functionality for executing commands. While it is descriptive for folks who are used to this type of feature, it may not be as appealing to those less familiar with it. As a non-developer user, I probably wouldn't think there is anything for me in a feature called "Command Palette."

On the other hand, I think "Wayfinder" has the potential to evoke a sense of curiosity, exploration, and discovery in more types of users. This is also important as the feature and its use cases evolve, so it's still my recommended option from a marketing perspective if we want to keep this and the longer-term aims in mind. I also agree with @eidolonnight in being mindful of where the name will primarily be used.

**Comment by alexstine** [comment]
Quick Actions or Quick Navigation? I've never heard the term Wayfinder and would not use something called a Wayfinder as an obvious means of navigation.

**Comment by aurooba** [comment]
We did hear Matías call it [a "Wayfinder tool" in the WordCamp Europe 2023 Keynote](https://youtu.be/7Nmz3IjtPh0?t=911), so at this point perhaps making arguments for it to be called something else _may_ be moot, I'm not sure. However, I was curious about the argument that a term using the word `command` would be less appealing to non-developers, so I wanted to document names around the internet for command palettes that I've seen in non-developer tools/services (to augment @rosswintle's documenting of tools and what they call this feature as well):

- Todoist calls it [a Command Menu](https://todoist.com/help/articles/navigate-faster-with-the-command-menu-and-your-keyboard)
- ClickUp calls it [a Command Center](https://help.clickup.com/hc/en-us/articles/6533695640343-Command-Center-overview)
- ConvertKit calls it [Quick Find](https://help.convertkit.com/en/articles/6384168-how-to-navigate-your-account-using-quick-find)
- Notion calls it [Search](https://www.notion.so/help/search#using-search)
- MacOS calls it [Spotlight](https://support.apple.com/en-ca/guide/mac-help/mchlp1008/mac)
- Superhuman calls it [Command](https://blog.superhuman.com/how-to-build-a-remarkable-command-palette/)
- Missive calls it [Command Bar](https://missiveapp.com/features/command-bar)
- Figma calls it [Quick Actions](https://help.figma.com/hc/en-us/articles/360040328653-Use-shortcuts-and-quick-actions#Quick_actions)
- Miro calls it [Command Palette](https://help.miro.com/hc/en-us/articles/10389262146706-Command-palette)
- Pika calls it [Search](https://pika.style/)
- Reflect calls it [a Command Palette](https://reflectchangelog.substack.com/p/new-release-command-palette)
- Obsidian calls it a [Command Palette](https://help.obsidian.md/Plugins/Command+palette)
- Cron calls it [a Command Menu](https://cronhq.notion.site/Cron-keyboard-shortcuts-a157029e2e78402f92254af3ef8094fd)

I'm sure there's more, these are just the ones I could think of, off the top of my head. I'm not sure I consider the argument that a name with the term `Command` would be less appealing to non-developers a very strong one.

That's not to say that WordPress shouldn't choose a different or unique name for this feature. However, then at this point I'm wondering what _kind_ of name is wanted: one that feels new and different or one that clearly communicates its purpose and easy to remember?

Those ideas don't have to be mutually exclusive, but going _against_ a fairly internet-wide informally established naming convention and understanding of a certain feature should have a solid reason behind it.

**Comment by eidolonnight** [comment]
> I've never heard the term Wayfinder and would not use something called a Wayfinder as an obvious means of navigation.

I believe this is intentional. In the same way that [Apple has trademarked "Spotlight"](https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html), there is a desire to give this feature a unique name that separates it from other things that may look like it but behave differently.

Hearing the ambitions for this from Matías and others, they go far beyond the functionality of the other tools referenced above. Using the commonly used names of other tools may help early on to communicate ideas, but continuing to use those names would misrepresent the functionality and potential to new users and developers.

**Comment by aurooba** [comment]
> Hearing the ambitions for this from Matías and others, they go far beyond the functionality of the other tools referenced above

That's interesting, @eidolonnight! Where can we find these ambitions documented that would help us understand the desire for such a name?

**Comment by justlevine** [comment]
> Hearing the ambitions for this from Matías and others, they go far beyond the functionality of the other tools referenced above

Echoing @aurooba, as nothing I've heard so far goes beyond VSCode's "Command Palette" (to mention an example I'm familiar with).

Further, the Google Search definition that pops up for "wayfinder" when I type it is:
> a sign, landmark, or other indicator used to assist people in navigating to a particular location.

The ambitions that I _have_ heard about so far do more than just navigate to different areas of the backend, but would allow users to take specific actions as well.

**Comment by eidolonnight** [comment]
> Where can we find these ambitions documented that would help us understand the desire for such a name?

In regard to the functionality both now and in the future, I'm referring to various conversations. I'll list some below.

- https://github.com/WordPress/gutenberg/issues/48457
- https://github.com/WordPress/gutenberg/pull/49681#issuecomment-1502480842
- https://github.com/WordPress/gutenberg/pull/50128
- [Command Center: Request for feedback](https://make.wordpress.org/core/2023/05/17/command-center-request-for-feedback/)
- [Let’s talk: WordPress Core & Artificial Intelligence](https://make.wordpress.org/core/2023/05/02/lets-talk-wordpress-core-artificial-intelligence/)

I've already seen numerous features mentioned: searching, navigating, performing actions, giving contextual recommendations, AI integration, and further extensibility through its API.

**Comment by remkus** [comment]
I understand the desire to brand this functionality into something with a unique name, but surely, we can find a better suiting one than wayfinder. As @justlevine highlights, the meaning of that has got nothing to do with what the functionality is hoping to solve. Now, and in the future. 

I actually think @aurooba's suggestions make way more sense as as whole.

**Comment by JosVelasco** [comment]
I have a new name proposal: `Command Finder`

So it can serve as a finder and unleash the power of actions. At least in Mac, It's triggered by the **command** key, and the `K` also reminds me of the word command.

**Comment by lesley-pizza** [comment]
TL;DR:
There are already many new changes coming to WordPress, so why don't we lessen the mental load for users and avoid new names and experiences where possible? Let's stick to convention for the most part so that the differentiated, big new features can really shine! 

The reasons for coming up with a new name:
1. the existing name is too technical
2. the existing name has militant connotations
3. Because of feedback left [here](https://make.wordpress.org/core/2023/05/17/command-center-request-for-feedback) - not sure which pieces of feedback suggest a new name is necessary
4. the existing name might not fully convey the potential and different use cases. 

I think these are great reasons to consider names that are alternative to "command center".

However, I'd like to suggest that instead of coming up with a new branded name, we simply use a **pre-existing term** that fulfils those criteria.

"Users spend most of their time on other sites. This means that users prefer your site to work the same way as all the other sites they already know." This is [Jakob's Law](https://lawsofux.com/jakobs-law/).

So unless there is a strong reason to force users to learn a new convention, we should simply pick a name that is already commonly in use.

To me, there are 3 reasons a new name might be justified:
1. this feature doesn't currently exist and there are no existing conventions. Hence, a completely new name must be invented. 
2. this feature is going to be a key differentiator and therefore it's worth branding for marketing purposes. As an example of what I mean by key differentiator - a user of a competing CMS decides to convert to WordPress due to the existence of this feature.
3. this feature is a huge part of the product, so much so that it's considered a mini product and therefore needs its own name. As an example, the block editor is a huge part of WordPress and warrants its own name.

Perhaps this feature satisfies one of these 3 reasons. Or perhaps there are other reasons a new name is justified. If so, I'd be very interested to learn more! 

As an example, I think it'd be really productive to have a discussion on just how big this feature really is. If we're committing time from the marketing team to really market this feature, then maybe having a branded name might be really worth it.

In the event that a new brand name isn't justified, I think Jakob's law should take precedence. And it'd be great if we could simply choose from the myriad of existing names already put forth by @lidialab, @aurooba and @rosswintle.

There are already many new changes coming to WordPress, so why don't we lessen the mental load for users and avoid new names and experiences where possible? Let's stick to convention for the most part so that the differentiated, big new features can really shine!

**Comment by bacoords** [comment]
Great points by all in this open discussion. One addition thought for consideration: 

For those of us attempting to find or create educational content for WordPress, the use of an off-topic codename for the block editor (Gutenberg) has made everything harder. Harder to label and describe our content, harder to search for existing resources, harder to educate our clients and users, etc. 

For those future developers and power users who might be searching for documentation (whether to simply use or else to extend this new feature), it's worth considering one of the more common names or phrases suggested above simply for discovery's sake, rather than a vague term geared towards marketing. What might sound fun at launch could end up being a pain point for future users once the 'new feature shine' wears off and the community is just trying to create educational resources for it.

**Comment by aurooba** [comment]
Thanks for sharing those links, @eidolonnight. I haven't seen anything in those links (or elsewhere) that provides a compelling argument for calling this feature `Wayfinder`, in fact, the arguments and functionality noted in those links make a strong case for choosing a name that emphasizes the act of _doing_ rather than _finding_, and the act of being able to contextually affect your place _and_ work within the WP-Admin, which is a much larger and wider feature definition than a term like `Wayfinder` implies.

@lesley-pizza has very eloquently put forth the arguments and rubric that should form the basis of evaluating not only _when_ a new name should be picked but also _what kind_ of name should be picked.

> As an example, I think it'd be really productive to have a discussion on just how big this feature really is. If we're committing time from the marketing team to really market this feature, then maybe having a branded name might be really worth it.
> – [from @lesley-pizza's comment](https://github.com/WordPress/gutenberg/issues/50925#issuecomment-1597496969)

I do get the sense that this feature is meant to be pretty big and to form an important part of whichever release it lands in, but even if that's the case, a branded name could still follow conventions we are all used to.

> the use of a off-topic codename for the block editor (Gutenberg) has made everything harder. Harder to label and describe our content, harder to search for existing resources, harder to educate our clients and users, etc.
> – [from @bacoords's comment](https://github.com/WordPress/gutenberg/issues/50925#issuecomment-1597598609)

This is _such_ an important piece of feedback from @bacoords, when thinking of the educational and support ecosystem that surrounds and uplifts the WordPress project. Discovery matters when it comes to adoption, and choosing a name that is at least a little connected to established conventions around the web would really really help, probably in more ways than one.

I want to iterate that I'm not against a new name by any means. I just want to understand why a new name might be needed and then if, in fact, it is needed, to pick a name that not only fits well but can be a catalyst for better adoption. 

`Wayfinder` sounds pretty but so far as I can tell, would be a pretty misleading term considering the the vision for this feature includes more than the ability to navigate and will hopefully be massively extensible.

**Comment by richtabor** [comment]
> On the other hand, I think "Wayfinder" has the potential to evoke a sense of curiosity, exploration, and discovery in more types of users. 

I agree with this sentiment.

> For those of us attempting to find or create educational content for WordPress, the use of an off-topic codename for the block editor (Gutenberg) has made everything harder. 

I don't think using a generic name will make it easier to reference in guides/videos — perhaps more difficult. 

> this feature is a huge part of the product, so much so that it's considered a mini product and therefore needs its own name. As an example, the block editor is a huge part of WordPress and warrants its own name.

Ironically, the editor will probably just be called the WordPress editor (remove "block") in time. The project does have a way of distilling "names" down to inherit functionality, which is important to note. 

> this feature is a huge part of the product

This tool will become increasingly more beneficial for folks who are particularly familiar with WordPress — especially as plugins start adding their own commands to it. 

I.e. It'd be neat to have little "iOS interactive widget" type micro experiences invoked from here. Perhaps a command reveals the latest WooCommerce sales trends within the experience, or suggests (and allows you to set) featured images.  

> Wayfinder sounds pretty but so far as I can tell, would be a pretty misleading term considering the the vision for this feature includes more than the ability to navigate and will hopefully be massively extensible.

I see Wayfinder as a helpful guide to help you navigate through WordPress. Not necessarily from point A to point B, but navigate as in actually helping you accomplish your mission. 

All that said, I'm not entirely leaning one way or the other, but I do like "Wayfinder" and "Command Palette".

**Comment by aurooba** [comment]
> > On the other hand, I think "Wayfinder" has the potential to evoke a sense of curiosity, exploration, and discovery in more types of users.
> 
> I agree with this sentiment.

I agree that the term `Wayfinder` evokes curiosity because it doesn't immediately communicate what the feature does. I also think that a term like `Wayfinder` negatively impacts discoverability.

> I don't think using a generic name will make it easier to reference in guides/videos — perhaps more difficult.

It's not about using a generic name, it's about using a _clear_ and consistent name that means what it says. 

> The project does have a way of distilling "names" down to inherit functionality, which is important to note.

Wouldn't it be wonderful if instead of a transition from a word like Gutenberg > Block Editor > WordPress Editor > the editor, we just had introduced it as "the new editor" and let it be, so (to @bacoords point) instead of searching for `keyword` + `gutenberg`, then `keyword` + `block editor` then `keyword` + `WordPress editor`, people could have just one term to reference things with? 

It'd be nice if this feature was named according to its functionality from the get go, if that's where things will eventually land as you say, @richtabor.

> I see Wayfinder as a helpful guide to help you navigate through WordPress. Not necessarily from point A to point B, but navigate as in actually helping you accomplish your mission.

That's an interesting way to put it. An almost personification, which is kind of a cool way to look at it. 

I believe a good name for this particular feature makes you want to use it rather than making you wonder what it is. A name that sounds like an indispensable tool you can grasp easily and integrate into your workflow, something you see immediate value in just from hearing its name, rather than one that makes you feel like it's something you'll have to learn and figure out.

**Comment by shilpahshah** [comment]
How about plain old 'Control Panel'?
'Panel' so that it is clear there's more than just a couple of of buttons / menus in there

**Comment by iansvo** [comment]
The term “Wayfinder” is very much associated with navigation and not with taking actions or giving commands. This feature’s purpose is to make it easier to run commands and move about the dashboard. 

A term like Wayfinder feels like marketing speak more than an actual feature name. Comaider the difference between saying: “use the Wayfinder” and “use the command palette”. In the later example, the meaning and purpose is immediately clear. So sure someone else may not know what a “command palette” is but I’d wager way more folks know what a command palette is than a random feature in me specific app called Wayfinder. 

I would ask that we focus on being easy to understand and to translate above trying to be clever with a name.

**Comment by audrasjb** [comment]
This has been said by a lot of people, but Wayfinder is pretty hard to translate in many languages. I suspect that there is a very small amount of language where this translates into a single term…

> From what I'm hearing, there does not seem to be any red flags with "Wayfinder," other than it's tricky to translate as it might not exist in some languages (I wonder if that opens the doors to treat it similarly to the word "plugin")

In my locale, "Plugin" is translated into "Extension" (since they "extend" WordPress and given "Plugin" is hard to understand for a non-english speaker who begin they WordPress journey :)
To me, using a term that is "tricky to translate" is a much of a red flag by nature when more than 50% of WordPress installs are not using American English.

Command Center sound great to me, and if it is much easier to translate for most languages, it's hard to me to understand why we should use a marketing term known to be hard to translate for more than 50% of the WP ecosystem :)

**Comment by miroslavglavic** [comment]
Wayfinder is something you'd find at the airport or a train station and ask for directions.

A Command Centre is where the decisions are made, in human terms: Your brain.

I don't really need directions when I am in the admin area of my website.

Also wayfinder is difficult to translate, you can't just do direct literal translation. It does not make sense in Spanish or Croatian by the way.

**Comment by dream-encode** [comment]
I don't have much to add to the discussion here, but I do agree with the majority here(and elsewhere) that feels "Wayfinder" is not a great choice.  I appreciate the difficulty in naming things, but this "Wayfinder" seems to be borne out of a need to brand the feature or otherwise give it some app-like name, rather that something functional and sematic.  

FWIW, I see some suggestions above that would work great, like "Command Center" or "Action Center".  These are much more descriptive and contextually accurate for the feature.  Also, as many have mentioned, these sorts of names exist for many things  outside of WordPress, and there's a familiarity for both uses and developers.

Also, translations need to be considered , and the fact that an existing plugin(https://wordpress.org/plugins/wayfinder/) has the exact name.  Not only that, it's a block editor plugin.  Sure the installs are not a huge number, but they claimed the name two years ago.

**Comment by spacedragn** [comment]
[Maggie Appleton calls this a Command Bar.](https://maggieappleton.com/command-bar)

I generally agree with the discourse around “Command” as it’s an established word for this UI component. The feature does what it says – executes commands. I’m against “Wayfinder” as it’s often synonymous with sticky Table of Contents components.

“Bar” is simpler than “Palette” which is abstract and easily typo’d. “Bar” is also more distinct than “Center.”

**Comment by Ren2049** [comment]
I like "Conductor", considering that in the future it will be mainly used to "conduct" AI. Also plays well with the Jazz musician theme of WP releases.

**Comment by aurooba** [comment]
Some of the suggested names so far and some thoughts:

- **Control Panel**, while I like the concept, I think Panel in the context of WordPress makes me think of those little sections in the Settings Sidebar, so I'm not sure it fits right.
- **Command Finder**, while both the words are good, combined they make you think of a tool that helps you _find_ a command rather than _use_ one.
- **Command Bar**, I think this is as simple as it gets. I think one of the reasons the original name had `Center` in it was because of this feature's capacity to expand and have a lot more going on in it (for example, [asking AI to suggest some designs](https://make.wordpress.org/core/2023/05/02/lets-talk-wordpress-core-artificial-intelligence/) for the current page and having it expand to show those options that you can select from, etc), and I do think we should be mindful of that, hence `Center` or `Palette` make more sense to me.
- **Conductor**, as far as "branded" names go, this is probably one of the better ones that also lends itself to some evocative imagery.

I was thinking about the screenshot shared in #51521, that shows how the name of the feature might be exposed in the UI, and personally, I still think something with the term `Command` or `Actions` makes the most sense both from a power user's perspective and also from a typical user's perspective, where one might see it in the UI and need a way to reference it.

Personally, I lean more towards the `Actions` term than anything else, because to @rmartinezduque's point, `action` is possibly the clearest and simplest term – accessible to everyone, devs and non-devs alike, and I think the concept of `WordPress Actions` or the `WordPress Actions [Palette/Center/Bar/]` is not only descriptive, translatable, and easy to understand but also feels like a nod to the extensible nature and history of WordPress, which as a dev, I like.

**Comment by lidialab** [comment]
Several interesting and useful points have been written. I agree with those who suggest keeping things simple. Both with those who want to avoid technical terms and with those who recommend keeping the familiar nomenclatures.
More than once, while I was reading the new comments, "Quick actions" pops up in my mind.

I like a lot (also as a Polyglots) "WordPress Actions".

**Comment by JosVelasco** [comment]
I'd love it was called ChatWP. Eventually, it could interact, not only execute commands or searches.

**Comment by paaljoachim** [comment]
Whatever the name the Command center feature ends up with.
The name should cover the vision as well as the usage of what the command center is used for now and will likely be used for in the future.

**Comment by courtneyr-dev** [comment]
> I'd love it was called ChatWP. Eventually, it could interact, not only execute commands or searches.

We might want to avoid that due to https://wpdocs.chat/.

**Comment by JosVelasco** [comment]
> > I'd love it was called ChatWP. Eventually, it could interact, not only execute commands or searches.
> 
> We might want to avoid that due to https://wpdocs.chat/.

If we found a name that most people like but is already taken, talking to the owners of the name to donate it to the community could be fantastic.

**Comment by annezazu** [comment]
Thank you everyone for the great discussion and feedback, including offering up possible names, sharing insights around translations, and doing research around the broader ecosystem of similar functionality in other products. To move this forward in line with WordPress 6.3's release cycle, I'm jumping in to wrap this discussion up and move forward after aligning with project leadership, reading through everyone's comments, and looking ahead at what's to come. 

**Let's move forward with Command Palette.**

Reasoning: easier to translate, consistent across other tooling outside of WordPress, matches current functionality, eases discoverability/understanding of value, and leans generic which matches the concerns raised here. Ultimately, we can always discuss renaming if the feature reaches a point of evolution outside of this initial name. As raised above, that would be more worth risking a unique name for than something that exists in other products and that ultimately we want people to quickly understand/find value in. Plus if we hold off on that name for the future, it can create a nice marketing push for something truly unique when/if the time comes. If folks have additional specific concerns around this naming, please speak up sooner rather than later. 

@youknowriad can you please ensure the naming is updated throughout the interface for 6.3? I can mainly think of these two areas:

Site View sidebar in the Site Editor:
<img width="410" alt="Screen Shot 2023-06-27 at 11 43 27 AM" src="https://github.com/WordPress/gutenberg/assets/26996883/a369dae4-c3d2-4539-ac84-e5076f477f71">

Title bar in the Site Editor:
<img width="465" alt="Screen Shot 2023-06-27 at 11 45 24 AM" src="https://github.com/WordPress/gutenberg/assets/26996883/5db1d6af-e011-480e-87f9-cfe6271d03b9">
 
For the latter, it likely touches on this issue https://github.com/WordPress/gutenberg/issues/51394 but noting as the other spot where this feature can be accessed. 

I will close this issue out fully when the above is handled with naming in the interface.

**Comment by priethor** [comment]
Thanks for summarizing this healthy discussion, @annezazu! I agree on moving forward with `Command palette` for all the reasons above 💯
